bouncy/package.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>console_bridge</name>
+  <version>:{version}</version>
+  <description>ROS-independent package for logging that seamlessly pipes into rosconsole/rosout for ROS-dependent packages.</description>
+  <maintainer email="steven@oopenrobotics.org">Steven! Ragnarök</maintainer>
```

Comment:
The email address seem invalid.

Reply:
nice catch! fixed in https://github.com/ros2-gbp/console_bridge-release/pull/1/commits/1d2840626d567e4db1e04065e4bf0ea2dfd46ff5 and https://github.com/ros2-gbp/console_bridge-release/pull/1/commits/e4dfdb52ca11edb5a4d1375e84e4193a95403d4b